src/ECCX08.cpp

```diff
@@ -440,6 +440,113 @@ int ECCX08Class::lock()
   return 1;
 }
 
+
+int ECCX08Class::beginHMAC(uint16_t keySlot)
+{
+  // HMAC implementation is only for ATECC608
+  long ver = version() & 0x0F00000;
+  if (ver != 0x0600000) {
+    return 0;
+  }
+
+  uint8_t status;
+
+  if (!wakeup()) {
+    return 0;
+  }
+
+  if (!sendCommand(0x47, 0x04, keySlot)) {
```

Comment:
We tend to stay away from magic numbers and instead have them as `#define`'s so that users can infer what they're doing. I don't know what `0x47` or `0x04` mean or do. Some descriptive macro would be useful.

Reply:
Thanks for pointing that out, I agree that descriptive macros would be useful. I sticked with the style already in place for now.